doc/source/changes.rst

```diff
@@ -14,6 +14,13 @@ Version 1.7.2 (CURRENT DEV)
 This is the current development version, not yet released.
 Changes thus far are relevant only for users of MapFish Print.
 
+Oereblex related changes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
It needs to be exactly the same length as the title, otherwise doc & package generation will fail